src/components/LanguageMenu/LanguageNativeSelect.tsx

```diff
@@ -25,16 +25,24 @@ export default function LanguageNativeSelect() {
     "& .MuiInputBase-input": {
       padding: 0,
       fontSize: theme.typography.caption.fontSize,
+      color: "black",
```

Comment:
`text.primary` should work as well and compose with the theme
```suggestion
      color: "text.primary",
```

Reply:
I also changed the color of the footer links:
https://github.com/pagopa/pagopa-checkout-fe/blob/bf65f6fe6c757ce9ce6cf46388b7314db9b5de62/src/components/commons/Footer.tsx#L36-L54